src/components/JitsiMeeting.tsx

```diff
@@ -0,0 +1,90 @@
+import { useEffect, useRef, useState } from "react";
+import { init } from "..";
+import { IJitsiMeetingProps } from "../types";
+import { getRoomName } from "../utils";
+
+/**
+ * Returns the JitsiMeeting Component with access to a custom External API
+ * to be used as-it-is in React projects
+ *
+ * @param {IJitsiMeetingProps} props
+ * @example
+  ```js
+      <JitsiMeeting
+        id="testId"
```

Comment:
What is this ID?

Reply:
I needed some uniqueness when testing the multiple instance scenario. Had it removed and generated an uuid in the function component body.